test/unit/Critic/CriticClientTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+if (!defined('APPROOT'))
+{
+    define('APPROOT', dirname(dirname(dirname(__DIR__))));
+}
+
+/**
+ * Unit tests for CriticClient
+ */
+class CriticClientTest extends PHPUnit_Framework_TestCase
```

Comment:
I thought you were renaming these to be `ClientTest` - otherwise the namespace will be `Talis\Critic\CriticClientTest`, which doesn't match the client.